src/protocol/thread/node.ts

```diff
@@ -215,7 +215,14 @@ export class NodeFront {
       this._pause
         .sendMessage(client.CSS.getAppliedRules, { node: this._object.objectId })
         .then(({ rules, data }) => {
-          this._rules = rules;
+          this._rules = [];
+          for (const rule of rules) {
+            if (
+              !this._rules.some(r => rule.rule === r.rule && rule.pseudoElement === r.pseudoElement)
+            ) {
+              this._rules.push(rule);
+            }
+          }
```

Comment:
could this be simplified with https://lodash.com/docs/4.17.15#uniqBy

```
this._rules = uniqBy(rules, rule => rule.pseudoElement)
```

Reply:
more like `uniqBy(rules, rule => `${rule.rule}|${rule.pseudoElement})`